### ISSUE TITLE
Swift 4

### ISSUE THREAD

**Issue by ilyavelilyaev** [onfido/onfido-ios-sdk#15]
Hey guys! Can I ask when we should expect Onfido compiled for swift 4?

**Comment by anuragajwani** [comment]
@ilyavelilyaev which version of Xcode are you using ? We are currently using Swift 3.2.2 which is bundled with Xcode 9.1. Swift 3.2.2 should be compatible with Swift 4.0.2.

**Comment by ilyavelilyaev** [comment]
@anuragajwani I am using Xcode 9.1, but I want to use latest ObjectMapper, which is 3.0 and you have it as dependency ~> 2.2, which is not swift 4 compatible

**Comment by anuragajwani** [comment]
@ilyavelilyaev the dependency should have been removed from the podspec. We will push a new release tomorrow morning to fix this issue.

Sorry for the inconvenience,
Anurag

**Comment by ilyavelilyaev** [comment]
@anuragajwani thank you so much! Looking forward

**Comment by anuragajwani** [comment]
@ilyavelilyaev I just tested manual installation with 5.0.1. If you are blocked currently by us then please remove Onfido from your Podfile and include the following dependencies

```
pod 'Alamofire', '< 4.5' # 4.5 has a bug
  pod 'MBProgressHUD', '~> 1.0.0'
  pod 'Swinject', '~> 2.0'
  pod 'SwiftyJSON', '~> 3.1
```
 
then embed the framework into your app (select your project xcodeproj file, select your app target and under the General tab drag and drop the framework into Embedded binaries section)
![screen shot 2017-11-14 at 16 32 23](https://user-images.githubusercontent.com/7071488/32791681-7f07f720-c959-11e7-814b-c7eec753f282.png)

To get the framework download the zip file from https://github.com/onfido/onfido-ios-sdk/tree/master/builds/5.0.1 and uncompress it; it will contain the framework.

Sorry once again for the inconvenience,
Anurag

**Comment by anuragajwani** [comment]
@ilyavelilyaev released [5.0.2](https://cocoapods.org/pods/Onfido) which does not require ObjectMapper anymore. Let me know if this resolves our issue please.

Thanks,
Anurag
